src/process/status.cr

```diff
@@ -25,6 +22,13 @@ class Process::Status
     Signal.from_value(signal_code)
   end
 
+  # Platform-specific exit status code, which usually contains either the exit code or a termination signal.
+  # The other `Process::Status` methods extract the values from `exit_status`.
+  @[Deprecated("Use `Process::Status#exit_code`")]
```

Comment:
I kept the structure the same as what I've seen in some of the other deprecation messages. I didn't want to make any assumptions like "Will be removed in 0.34" or whatever.

Reply:
Also, I moved the getter to a method for the deprecation. I wasn't sure if that annotation works on the getter/setter/property macros or not.

Reply:
IIRC it should